Firmware/printer.cfg

```diff
@@ -253,12 +282,12 @@ gcode:
    G1 E-40 F1800                  ; retract some, but not too much or it will jam
    M82                            ; set extruder to absolute
 
-# Sensor Types
-#   "EPCOS 100K B57560G104F"
-#   "ATC Semitec 104GT-2"
-#   "NTC 100K beta 3950" (Keenovo Heater Pad)
-#   "Honeywell 100K 135-104LAG-J01"
-#   "NTC 100K MGB18-104F39050L32"
-#   "AD595"
-#   "PT100 INA826"
-
+## 	Common Temperature Sensors
```

Comment:
id reword this back to sensor types, klipper supports a certain list of sensor types "common" makes it sound like there may be others that would work here but i dont believe that is the case.